model/src/main/java/org/cbioportal/model/CancerStudy.java

```diff
@@ -2,13 +2,101 @@
 
 import org.cbioportal.model.summary.CancerStudySummary;
 
-import java.io.Serializable;
-import java.util.Date;
 
 public class CancerStudy extends CancerStudySummary {
 
     private TypeOfCancer typeOfCancer;
     private Integer sampleCount;
+    private String stableID;
```

Comment:
this is a class representing a cancer study, and cancer study doesn't have a stable id, so stable id of what?